operators/src/source/ogr_source/mod.rs

```diff
@@ -610,6 +610,64 @@ impl FeaturesProvider<'_> {
         Ok(())
     }
 
+    ///  Creates a time filter string.
+    fn create_time_filter_string(
+        time_type: OgrSourceDatasetTimeType,
+        mut time_interval: TimeInterval,
+        driver: &str,
+    ) -> Option<String> {
+        match driver {
+            "PostgreSQL" => {
+                let postgres_range: TimeInterval = // https://www.postgresql.org/docs/current/datatype-datetime.html, capped by max supported TimeInstance value
+                    TimeInterval::new_unchecked(-210_895_056_000_000, 8_210_266_876_799_999);
+
+                if !postgres_range.contains(&time_interval) {
+                    return None;
+                }
+
+                time_interval = postgres_range.intersect(&time_interval)?;
+            }
+            _ => return None,
+        }
+
+        let t_start = time_interval.start();
+        let t_end = time_interval.end();
+        let comp_end = if time_interval.is_instant() {
+            "<="
+        } else {
+            "<"
+        };
+
+        #[allow(clippy::match_same_arms)]
+        match time_type {
+            OgrSourceDatasetTimeType::None => None,
+            OgrSourceDatasetTimeType::Start {
+                start_field,
+                start_format: _,
+                duration,
+            } => {
+                match duration {
+                    OgrSourceDurationSpec::Infinite => {
+                        Some(format!(r#""{start_field}" < '{t_end}'"#))
+                    }
+                    OgrSourceDurationSpec::Zero => Some(format!(
+                        r#""{start_field}" {comp_end} '{t_end}' AND "{start_field}" >= '{t_start}'"#
+                    )),
+                    OgrSourceDurationSpec::Value(_) => None, // TODO
```

Comment:
throw an error then or do you want to implement it still?

Reply:
Same as above. I would not throw an error. Returning None here simply means that we defer the time filtering to the GE instead of already filtering with PG. This case does not occur with the GBIF data and is potentially complicated since the addition of the start field and the duration value must be possible